src/WordPress/WordPressAPI.php

```diff
@@ -8,6 +8,7 @@
 class WordPressAPI implements IntegrationAPIInterface
 {
     private $dataStore;
+    private static $domainName;
```

Comment:
Why is this static?  This class has a constructor and isn't intended to be used statically.